app/src/main/java/com/datarockets/mnchkn/injection/modules/ApplicationModule.kt

```diff
@@ -5,25 +5,22 @@ import android.content.Context
 import com.crashlytics.android.Crashlytics
 import com.crashlytics.android.core.CrashlyticsCore
 import com.datarockets.mnchkn.BuildConfig
-
 import com.datarockets.mnchkn.injection.ApplicationContext
-import com.mixpanel.android.mpmetrics.MixpanelAPI
-
 import dagger.Module
 import dagger.Provides
 
 @Module
-class ApplicationModule(protected val mApplication: Application) {
+class ApplicationModule(private val application: Application) {
 
     @Provides
     fun providesApplication(): Application {
-        return mApplication
+        return application
     }
 
```

Comment:
@vshkl This is the place where you can initialize RoomDatabase and then pass the result as a parameter for providing `gameDao` and `playerDao`. Please, consider about doing this.